### ISSUE TITLE
Give the API a sort of 'homepage' for Cloudflare proxy and SSL support.

### ISSUE THREAD

**Issue by Thomaswill1** [SnailyCAD/snaily-cadv4#143]
### Describe the feature

Please give the API a 'homepage'

Currently, when trying to visit the API URL, it returns with a ['refused to connect'](https://cdn.discordapp.com/attachments/625715899353268254/908384770055094342/unknown.png) error, and when Cloudflare servers see this, [it gives the server its 'catchall' response code of 'Error 520'](https://cdn.discordapp.com/attachments/625715899353268254/908384584482312242/unknown.png) which then doesn't allow for proxying over Cloudflare or allow SSL. When visiting the API URL, it can at least display a '200 OK' response.

Thanks.

TLDR: When visiting `http://99.99.99.99:8080` (with nothing else following that URL), return a displayed webpage, instead of `refused to connect`, to allow Cloudflare proxying and SSL.

**Comment by Dev-CasperTheGhost** [comment]
Will work on this soon!
